tests/settings/test_settings_sign_out_and_quit.py

```diff
@@ -10,10 +10,8 @@
 
 @pytest.mark.critical
 @allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703010', 'Settings - Sign out & Quit')
-# TODO: Experimental link for testing multiple references in test rail report by nightly job. Has to be removed!
```

Comment:
this reminder is still actual, because we need to link it to correct tests in case of success

Reply:
Ah, sorry, will put it back